Lombiq.Tests.UI/UITestBase.cs

```diff
@@ -45,7 +45,7 @@ public abstract class UITestBase
 
             if (timeoutTask.IsCompleted)
             {
-                throw new TimeoutException($"The time allotted for the test ({timeout}) was exceeded.");
+                throw new TimeoutException($"The time allowed for the test ({timeout}) was exceeded.");
```

Comment:
Revert this change. The phrase "the time allotted" is grammatically correct and appropriate for this specific context.
![image](https://github.com/user-attachments/assets/b9bb7bc8-2eec-4d67-8766-115aa9337921)

Reply:
Oh sorry 😅